src/components/SearchRecommendations/SearchRecommendationItem.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import SearchRecommendationItem from "./SearchRecommendationItem";
+
+export default {
+  title: "Components/SearchRecommendations",
+  component: SearchRecommendationItem,
+
```

Comment:
The stories are not displaying the images. Please see https://imgur.com/qdjCmUx

Reply:
done